lib/IDE/SyntaxModel.cpp

```diff
@@ -1038,9 +1038,9 @@ class IdRefWalker : public ASTWalker {
         return { true, E };
       if (DRE->getRefKind() != DeclRefKind::Ordinary)
         return { true, E };
-      // TODO: Handle special names
-      if (!Fn(CharSourceRange(DRE->getSourceRange().Start,
-                              DRE->getName().getBaseIdentifier().getLength())))
+      if (!Fn(CharSourceRange(
+              DRE->getSourceRange().Start,
+              DRE->getName().getBaseName().userFacingStr().size())))
```

Comment:
Note for @nathawes and @bitjammer: this is not a good way to measure the length of an identifier because it won't handle the backtick-escaping syntax. Using the lexer is the right way to do it.